fhir_transformer/src/analyze/analyzer.py

```diff
@@ -22,13 +23,31 @@ def __init__(self):
         # storing here, for instance?
         self.analyses = {}
         self._cur_analysis = Analysis()
+        self.last_updated_at = time.time()
```

Comment:
I think this last_updated at should belong to each Analyses and not to the Analyzer

Reply:
Hum that's a good point. Either we have a dict with the key as resource_id and value as last_updated timestamp. Or we add the last_updated_at in the Analysis object. 
What do you think is the best ?